### ISSUE TITLE
Add citations

### ISSUE THREAD

**Issue by ALGBBSH** [normen/whatscli#47]
En first moment, i would like to thank him your interesant program. I love it.

I would like that you add citations the messages, i miss this function. 

Thank you very much.

**Comment by normen** [comment]
Yeah, it's a bit annoying to implement though, I'll look into it. Keeping this open.

**Comment by ALGBBSH** [comment]
If I would  know golang, I would try implement it. But, it's impossible for me.  I image this:

Control +w
q (quoting)
and write the message.

The whatsapi is perfect for me. The only missing function is to quote messages, I use it often.

Thanks for publicate this program, norman, for your generosity.

**Comment by normen** [comment]
Right, I pushed the dev branch where I made those changes, thing is I don't exactly know how to compose the quoted message, seems some other people have issues with this as well, see here: https://github.com/Rhymen/go-whatsapp/pull/217

Right now the WhatsApp server responds with 400 so theres still some issue with the data. See sendQuotedMessage in session_manager.go - seems like I need to pull some other/more info from the old message when composing the new one. Thats what I meant by "annoying".
